po/pt.po

```diff
@@ -7723,7 +7723,7 @@ msgstr "Contabilidade de finanças pessoais e micro empresas."
 #. contributors.
 #: gnucash/gnome-utils/gnc-main-window.cpp:5268
 msgid "translator-credits"
-msgstr "Pedro Albuquerque <pmra@gmx.com>"
+msgstr "Pedro Albuquerque <pmra@gmx.com>, 2024"
```

Comment:
@pmraps:
1. My calendar still shows 2023 ;-)
2. IMHO you should display the year range of your contributions
3. You got suport this year by Hugo Carvalho:
https://github.com/Gnucash/gnucash/blob/bf460b0040f8c9837cca198a9daba83a91c6c35b/po/pt.po#L4C1-L6C50
IMHO you should mention him, too.